Say Their Names/Source/View/Home/Person/PersonCell.swift

```diff
@@ -9,10 +9,26 @@
 import UIKit
 
 class PersonCell: UICollectionViewCell {
+    static let personIdentifier = "personCell"
 
+    @IBOutlet var personPhotoImageView: UIImageView!
+    @IBOutlet var nameLabel: UILabel!
+    @IBOutlet var dateLabel: UILabel!
+    
     override func awakeFromNib() {
         super.awakeFromNib()
         // Initialization code
     }
+    
+    func configure(with person: Person) {
+        let df = DateFormatter()
+        df.dateFormat = "dd.MM.yyyy"
+        
+        personPhotoImageView.image = #imageLiteral(resourceName: "man-in-red-jacket-1681010")
```

Comment:
While we're making changes, let's replace the `imageLiteral` as it is susceptible to crashes
 ```suggestion
        personPhotoImageView.image = UIImage(named: "man-in-red-jacket-1681010")
```